common/src/main/java/de/maxhenkel/voicechat/config/ClientConfig.java

```diff
@@ -163,48 +163,48 @@ public ClientConfig(ConfigBuilder builder) {
                 );
         denoiser = builder
                 .booleanEntry("denoiser", false,
-                        "If noise cancellation should be enabled"
+                        "Specifies whether noise suppression should be enabled"
                 );
         runLocalServer = builder
                 .booleanEntry("run_local_server", true,
-                        "If voice chat should work in singleplayer/LAN worlds"
+                        "Controls whether the voice chat should work in singleplayer or in worlds shared over LAN"
                 );
         javaMicrophoneImplementation = builder
                 .booleanEntry("java_microphone_implementation", Platform.isMac(),
-                        "Whether to use the Java implementation of microphone capturing instead of OpenAL"
+                        "Controls whether to use the Java implementation of microphone capture instead of OpenAL"
```

Comment:
Why change this? Now you used "Controls whether" again. Again, no need to change this.